src/transaction/log_manager.c

```diff
@@ -4450,6 +4463,38 @@ log_append_donetime_internal (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA
   LSA_COPY (eot_lsa, &lsa);
 }
 
+static void
+log_append_group_commit (THREAD_ENTRY * thread_p, LOG_TDES * tdes, INT64 stream_pos, const tx_group & group,
+			 LOG_LSA * commit_lsa)
+{
+  LOG_PRIOR_NODE *node;
+  LOG_LSA lsa;
+
+  commit_lsa->pageid = NULL_PAGEID;
+  commit_lsa->offset = NULL_OFFSET;
+
+  // *INDENT-OFF*
+  std::vector<gc_tran_info> v;
```

Comment:
1. the usual approach is to create a buffer, pack required info and pass packed buffer to prior_lsa_alloc_and_copy_data.
2. for commit with postpone, also postpone lsa is needed.

Reply:
1: since log file doesn't need to be read by another host, I think is safe this "reinterpret cast"; please notice there are other legacy log records using the same approach.